Configuration.props

```diff
@@ -84,6 +84,7 @@
     <AllSupportedTargetAndroidAbis>$(AllSupported32BitTargetAndroidAbis);$(AllSupported64BitTargetAndroidAbis)</AllSupportedTargetAndroidAbis>
     <XABuildToolsVersion>28</XABuildToolsVersion>
     <XABuildToolsFolder Condition="'$(XABuildToolsFolder)' == ''">28.0.0</XABuildToolsFolder>
+    <XAPlatformToolsVersion>28.0.0</XAPlatformToolsVersion>
```

Comment:
Should this be `Conditional`/overridable, as `$(XABuildToolsFolder)` is?

Reply:
That's true. Though there are other properties without Condition, they are added in some need perhaps? We'll do so too.